packages/contractkit/src/wrappers/SortedOracles.ts

```diff
@@ -105,6 +109,12 @@ export class SortedOraclesWrapper extends BaseWrapper<SortedOracles> {
     valueToBigNumber
   )
 
+  removeExpiredReports = proxySend(
+    this.kit,
+    this.contract.methods.removeExpiredReports,
+    tupleParser(stringIdentity, valueToString)
+  )
+
```

Comment:
This function was added in another PR that merged today: https://github.com/celo-org/celo-monorepo/pull/3367

The main difference is that it takes the name of the token, rather than the address, to be consistent with the rest of the functions here.

Reply:
Okay cool, I'll use that one instead